edx_solutions_api_integration/courses/views.py

```diff
@@ -1124,6 +1124,50 @@ def get_queryset(self):
         return users
 
 
+class CoursesUsersPassedList(SecureAPIView):
+    """
+    **Use Case**
+
+        CoursesUsersPassedList returns a collection of users passed in the course.
+
+        You also use CoursesUsersPassedList to enroll a new user in the course.
+
+    **Example Requests**
+
+          GET /api/courses/{course_id}/users/passed
+
+    **GET Response Values**
+
+        * results: The collection of user_ids in the course.
+        * GET supports filtering of user by organization(s), groups
+         * To get users paased in a course and are also member of organization
+         ```/api/courses/{course_id}/users?organizations={organization_id}```
+         * organizations filter can be a single id or multiple ids separated by comma
+         ```/api/courses/{course_id}/users?organizations={organization_id1},{organization_id2}```
+         * To get users paased in a course and also member of specific groups
+         ```/api/courses/{course_id}/users?groups={group_id1},{group_id2}```
+    """
+
+    def get(self, request, course_id):  # pylint: disable=W0613
```

Comment:
nit: use pylint disable statement instead of code. i.e `# pylint: disable=unused-argument`